Tools/GNUMake/comps/hip.mak

```diff
@@ -40,7 +40,12 @@ HIPCC_FLAGS += --amdgpu-target=$(AMD_ARCH)
 
 CXXFLAGS += $(HIPCC_FLAGS)
 
-# =============================================================================================
+# add fopenmp targetting the gnu library (assuming CXX wraps gnu)
```

Comment:
What does "CXX wraps gnu" mean?  CXX is set to hipcc here.  hipcc is a perl script that will call amd's clang compiler.

Reply:
This comment was a distinction between the runtime environment, such as Cray, clang, or gnu. Pages 11-15 of this presentation addresses the different options: https://www.olcf.ornl.gov/wp-content/uploads/2019/11/Cray-Compiler-and-OpenMP.pdf

Reply:
I understand the effect of the flag.  It's just I think the comment inside () is confusing.

Reply:
Should I update the comment to clarify about the runtime libraries? or would you rather I just delete the confusing comment inside ()

Reply:
I think the first part of the comments is already very clear.  Maybe we can remove the part inside ().